bessctl/commands.py

```diff
@@ -1532,11 +1532,12 @@ def tcpdump_module(cli, module_name, direction, gate, opts):
             pass
 
 
-@cmd('track ENABLE_DISABLE [MODULE] [DIRECTION] [GATE]',
+@cmd('track ENABLE_DISABLE [DIRECTION] [MODULE] [GATE]',
      'Count the packets and batches on a gate')
```

Comment:
I understand fixing auto-complete, but I am a little uncomfortable with the reordering here. I understand that in your use case you want to vary direction without affecting the module used, but this seems far from general to me.